fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/bottomsheet/BottomSheet.kt

```diff
@@ -348,6 +355,13 @@ fun BottomSheet(
                                 }
                                 true
                             }
+                        } else if (!enableSwipeDismiss) {
```

Comment:
enableSwipeDismiss check should be applied on dismiss not on collapse to avoid sheet goes to invisible state.

Reply:
Done